lib/simulator-xcode-6.js

```diff
@@ -227,8 +229,38 @@ class SimulatorXcode6 {
     await killAllSimulators();
   }
 
+  async endSimulatorDaemon () {
+    // Looks for launchd daemons corresponding to the sim udid and tries to stop them cleanly
+    // This prevents xcrun simctl erase hangs.
+    log.debug(`Killing any simulator daemons for ${this.udid}`);
+    try {
+      let stopCmd = `launchctl list | grep ${this.udid} | cut -f 3 | xargs -n 1 launchctl stop`;
+      await exec('bash', ['-c', stopCmd]);
+    } catch (err) {
+      log.warn('Could not stop simulator daemons, carrying on anyway!');
+    }
+    try {
+      let removeCmd = `launchctl list | grep ${this.udid} | cut -f 3 | xargs -n 1 launchctl remove`;
+      await exec('bash', ['-c', removeCmd]);
+    } catch (err) {
+      log.warn('Could not remove simulator daemons, carrying on anyway!');
+    }
+    let pid;
```

Comment:
How is this `pid` being set?I see it being used in the `catch` block but nowhere does it seem to be given a value.